src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_PRICE = 40;
+  const DISCONT_1 = 20;
+  const DISCONT_2 = 50;
+  const TERM_SHORT = 3;
+  const TERN_LONG = 7;
+
+  if (days < TERM_SHORT) {
+    return DAY_PRICE * days;
+  }
+
+  if ((days >= TERM_SHORT) && (days < TERN_LONG)) {
```

Comment:
You don't need to check 'days >= TERM_SHORT' here because it's already covered in previous if statement. You can simplify this by just checking 'days < TERM_LONG'.